cime_config/config_archive.xml

```diff
@@ -129,9 +128,8 @@
     <test_file_names>
       <tfile disposition="copy">rpointer.glc</tfile>
       <tfile disposition="copy">casename.mali.rst.1976-01-01_00000.nc</tfile>
-      <tfile disposition="copy">casename.mali.rst.am.timeSeriesStatsMonthly.1976-01-01_00000.nc</tfile>
-      <tfile disposition="move">casename.mali.hist.am.globalStats.1976-01-01.nc</tfile>
-      <tfile disposition="move">casename.mali.hist.am.highFrequencyOutput.1976-01-01_00.00.00.nc</tfile>
+      <tfile disposition="move">casename.mali.hist.1976-01-01_00000.nc</tfile>
```

Comment:
This was just templated from the other MPAS component sections of this same file. I don't think the 1976 date matters and nothing I changed here changed anything w.r.t. how the current testing infrastructure treats MPAS (MALI) hist files (basically, it ignore them). I have another set of communications on this (see the "Notes from Jason Boutte" section of [this page](https://acme-climate.atlassian.net/wiki/spaces/PSC/pages/3687612417/Support+for+E3SM+configurations+with+a+dynamic+land+ice+MALI+component)) but my thinking at the moment is that maybe we leave that out of this PR so that we can experiment with and fine tune what's needed here as part of a future PR.

Reply:
Ok, thanks - either way is fine with me